core/src/Kokkos_Complex.hpp

```diff
@@ -829,4 +829,13 @@ struct reduction_identity<Kokkos::complex<T> > {
 
 } // namespace Kokkos
 
+namespace std
+{
+  template <typename Number>
+  Number abs( Kokkos::complex<Number> n )
+  {
+    return std::sqrt(n.real()*n.real()+n.imag()*n.imag());
```

Comment:
This may overflow unnecessarily.  You likely will not need to implement this (see above), but if you do, use `std::hypot`: https://en.cppreference.com/w/cpp/numeric/math/hypot

Reply:
I didn't realize that there is `Kokkos::complex` already, but it uses the same implementation. Let me change that here instead.